release-notes/release-notes52.md

```diff
@@ -125,10 +146,37 @@ The {{site.productname}} 5.2 release includes **Spellchecker Pro** 2.0.
 
 {{site.productname}} 5.2 provides fixes for the following bugs:
 
--
+- Fixed the incorrect parsing of malformed/bogus HTML comments.
+- Fixed the `quickbars` selection toolbar appearing on non-editable elements.
+- Fixed a bug with alignment toolbar buttons sometimes not changing state correctly.
+- Fixed the `codesample` toolbar button not toggling when selecting code samples other than HTML.
+- Fixed content incorrectly scrolling to the top or bottom when pressing enter when the content was already in view.
+- Fixed the `scrollIntoView` event potentially hiding elements behind the toolbar.
```

Comment:
`scrollIntoView` isn't an event, it's an editor API.

Reply:
fixed